src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
but there is a `.then` method:
![screen shot 2017-08-13 at 4 15 07 am](https://user-images.githubusercontent.com/154732/29249104-065ab53e-7fde-11e7-80c5-84f3de3c3258.png)

so thats not an issue with the babel plugin. if `.then` doesn't work on `import()`, i.e: `import().then`, then we have a problem. 

and if that's a problem, use pass resolveWeak to Promise.resolve for now, and I'll "resolve" it for the plugin lol